### DISCUSSION TITLE
Making the Website more user friendly for newbies to Open Source design

### DISCUSSION THREAD

**Discussion by Erioldoesdesign** [opensourcedesign/opensourcedesign.github.io#440] [Ideas]
In relation to this issue: https://github.com/opensourcedesign/opensourcedesign.github.io/issues/414

Any comments or feedback on how to improve the website opensourcedesign.net can be added here or in our discourse forum: https://discourse.opensourcedesign.net/c/meta/18

**Comment by jdittrich** [comment]
Maybe we should define some scenarios and then see how well the current website works in regards to them?

* I sometimes direct people who are in "how can usability of FOSS projects be improved" to the website. Would they find out what opensourcedesign does (community organization, meetings, forum) and what it does not (consulting FOSS projects, write own FOSS software)?
* People who are designers in open source projects and want to connect to other people doing similar work?
* People who want to post a design related job in an open source projects
